Source/ScreenShotUtils.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "Config.h"
+#include "PS2VM.h"
+
+class CScreenShotUtils : public Framework::CConfig
+{
+public:
+    typedef std::function<void(int, const char*)>        Callback;
+
+    static void                             TriggerGetSceenshot(CPS2VM*, Callback);
+
+private:
+    static CConfig::PathType                GetScreenShotDirectoryPath();
+    static CConfig::PathType                GetGameScreenShotDirectoryPath(std::string gameID);
```

Comment:
Is there any need to sort screen shots in separate directories for every game? I think it should be sufficient to prepend the game's id in the screen shot's file name.